### DISCUSSION TITLE
Importação do Json para o Mockoon

### DISCUSSION THREAD

**Discussion by ClaudineiMonteiro** [balta-io/forum#4] [Tira Dúvidas] [answered]
Estou tentando reproduzir a aula "Configurando o Backend" do curso 7187, onde baixei a ultima versão Mockoon para windows e estou tentando importar o arquivo Json da pasta assets, poréma importação não acontece e gera o seguinte erro:
Error while importing enviroment from OpenApi forms: SystaxError: C:\Couses\Angular\7187-master\assets\7187_2022-02-05.json is not a valid OpenApi definition.
Isto está impossibilitando de reproduzir as demais aulas devido ao retorno das api's não não existirem.

Observação: É muito importante pra mim, pois estou que concluir este curso até 09/16/2022, por causa da empresa.
Vai em anexo a imagem
![Balta_7187_Configurando_Backend](https://user-images.githubusercontent.com/16354515/189907925-a2cb26d1-6b63-44cc-98a0-21ea8caf209f.png)
@andrebaltieri

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @ClaudineiMonteiro, poderia por favor tentar a importação com este arquivo aqui:
https://github.com/balta-io/7187/blob/master/assets/7187_2022-09-13.json

**Comment by ClaudineiMonteiro** [comment]
Mesmo erro:
***@***.***

**Reply by ClaudineiMonteiro** [reply]
Mesmo erro
![Balta_7187_Configurando_Backend_2022-09-13_0227pm](https://user-images.githubusercontent.com/16354515/189981497-b1004838-b148-4ea6-80d0-04ea54e1e3ae.png)

**Comment by andrebaltieri** [comment]
@ClaudineiMonteiro tenta abrir o arquivo ao invés de importar...

<img width="177" alt="Screen Shot 2022-09-13 at 16 00 29" src="https://user-images.githubusercontent.com/965305/189987828-44dd7524-7c28-4b74-a26d-9a4653c16611.png">

Aqui funcionou:
<img width="1439" alt="Screen Shot 2022-09-13 at 16 01 38" src="https://user-images.githubusercontent.com/965305/189987848-ff7afd8e-bfc2-4ac4-a902-3054a2999484.png">

**Reply by ClaudineiMonteiro** [reply]
Mesmo não conseguindo importar o arquivo eu abri o arquivo e montei as requisições baseadas no arquivo e agora entendi o esquema e estou conseguindo trabalhar.
Foi resolvido muito obrigado

**Comment by ClaudineiMonteiro** [comment]
Consegui abrir o arquivo json e entender o funcionamento, assim montei as requisições e incluí o conteúdo nas mesma e agora estou conseguindo continuar, muito obrigado pela ajuda.

**Reply by ClaudineiMonteiro** [reply]
O unico prolema é que as imagens não estão mais disponiveis.
![Balta_7187_Configurando_Backend_2022-09-13_0355pm](https://user-images.githubusercontent.com/16354515/189997110-24d705a4-bca1-46f1-b801-2301c3fdab3d.png)

**Reply by andrebaltieri** [reply]
@ClaudineiMonteiro você pode usar algum serviço de placehold de imagem como este -> https://placeholder.com/

**Reply by ClaudineiMonteiro** [reply]
Obrigado é o que já estou fazendo. Muito obrigado mesmo pelas respostas este serviço aqui está ótimo.
